udata/templates/dataset/resource/list-item.html

```diff
@@ -1,12 +1,17 @@
 {% set resource_format = resource.format|trim|lower or 'data' %}
-<div id="resource-{{resource.id}}" class="list-group-item"
+<div id="resource{% if resource.from_community %}-community{% endif %}-{{resource.id}}" class="list-group-item"
```

Comment:
Hum, this distinction comes from this.
As ressources IDs (including community ressources) are unique and behavior is stricly the same, is this really necessary to have distinct ID pattern ?

Reply:
I need the distinction to use `showResource` that expects a `isCommunity` parameter, apparently in order to fetch the right resource in the JSON-LD.

The only alternative I can think of right now is to try to find the resource by id in both  `dataset[resources]` and `dataset[communityResources]`. I'm not sure it's a better alternative.